apps/api/src/app/events/services/workflow.queue.service.ts

```diff
@@ -85,6 +89,9 @@ export class WorkflowQueueService {
         events: job.digest.events,
       })
     );
+    if (job.presend === true) {
+      return;
+    }
```

Comment:
So looks that we are creating the message here above, could explain a bit on this phase? I was thinking that if this was present, it shouldn't create it. 

Or does it's just the jpb that happens before the digest will process it?

Reply:
Got ya. Thanks for the explenation